php/libraries/NDB_Caller.class.inc

```diff
@@ -467,14 +467,14 @@ class NDB_Caller
             header("Location: $redirectToOnSuccess");
             return "";
         }
-        $user =& User::singleton();
+        $user = User::singleton();
 
         if ($_REQUEST['sessionID']) {
-            $timepoint =& TimePoint::singleton($_REQUEST['sessionID']);
```

Comment:
This should be kept as a returned by reference object since the Timepoint class keeps a reference to it as well.